src/sentry/api/serializers/models/release.py

```diff
@@ -46,6 +46,164 @@ def expose_version_info(info):
     }
 
 
+def _expose_health_data(data):
```

Comment:
These functions were extracted from ReleaseSerializer so they can be used by both seriailzers